dynamics-nav-app/change-role.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Change the Role Center
+title: Description on How To Change the Role Center
```

Comment:
On or of?

Reply:
I had meant "on" but it sounds better "of". I'll fix it. Thanks!